PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,17 +19,18 @@
 - [ ] YES
 - [ ] NO
 
-### Does this PR introduce a breaking change? Please see definition of breaking change below
+### Does this PR introduce a breaking change? Please see few examples of breaking change below.
 
 - [ ] YES - please specify
 - [ ] NO
 
 > **Examples of breaking changes:**
 > - **causes app or operator downtime**
 > - modifies, deletes or moves the name of a job or instance group
-> - changes the name of a credentials
+> - modifies the name or deletes a property of a job or instance group 
+> - changes the name of credentials
 > - requires out-of-band manual intervention on the part of the operator
-> - removes Ops-file (e.g. from `./operations/` or `./operations/experimental` folders)
+> - removes Ops-files from `./operations/` or `./operations/experimental` folders
```

Comment:
Folders that we care about for breaking changes:
drwxr-xr-x  6 pivotal staff   192 May  8 16:44 addons
drwxr-xr-x 13 pivotal staff   416 Jul 25 14:54 backup-and-restore
drwxr-xr-x 10 pivotal staff   320 Aug  1 15:08 bits-service
drwxr-xr-x 43 pivotal staff  1376 Aug  1 15:08 experimental

Folders that we do not care about:
drwxr-xr-x  5 pivotal staff   160 Jul  9 15:49 community
drwxr-xr-x 18 pivotal staff   576 Mar  7 16:01 example-vars-files
drwxr-xr-x 17 pivotal staff   544 Aug  1 15:08 test
drwxr-xr-x  3 pivotal staff    96 Jan  7  2019 workaround

Reply:
@Syerram Were you going to update the template to reference the directories we identified above?